examples/platform/silabs/efr32/sl_custom_board.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * This file is used to set the pins for the SPI for Custom boards
+ * The SPI pins are defined in the file
+ *
+ *  !!!! MODIFY THIS FILE TO THE CORRECT PINS !!!!
+ */
+
+#ifndef _CUSTOM_BOARD_H_
+#define _CUSTOM_BOARD_H_
+
+#define WAKE_INDICATOR_PIN PIN(D, 2)
+#ifdef RS911X_WIFI
+// SPI ports and pins
+#define EUS1MOSI_PORT gpioPortC
+#define EUS1MOSI_PIN 1
+#define EUS1MISO_PORT gpioPortC
+#define EUS1MISO_PIN 2
+#define EUS1SCLK_PORT gpioPortC
+#define EUS1SCLK_PIN 3
+#define EUS1CS_PORT gpioPortC
+#define EUS1CS_PIN 0
+
+#define MY_USART EUSART1
+#define MY_USART_CLOCK cmuClock_EUSART1
+#define MY_USART_TX_SIGNAL dmadrvPeripheralSignal_EUSART1_TXBL
+#define MY_USART_RX_SIGNAL dmadrvPeripheralSignal_EUSART1_RXDATAV
+
+#define WFX_RESET_PIN PIN(A, 6)
+#define WFX_INTERRUPT_PIN PIN(A, 7)
+#ifdef EXP_BOARD
```

Comment:
Are those different pin defines both for Exp hdr 7?

Reply:
no, will remove the comment for the non expansion board